hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -107,7 +107,7 @@ static String getSparkShellCommand(String commandFile) {
         .append(" --master local[2] --driver-class-path ").append(HADOOP_CONF_DIR)
         .append(
             " --conf spark.sql.hive.convertMetastoreParquet=false --deploy-mode client  --driver-memory 1G --executor-memory 1G --num-executors 1 ")
-        .append(" --packages com.databricks:spark-avro_2.11:4.0.0 ").append(" -i ").append(commandFile).toString();
+        .append(" --packages org.apache.spark:spark-avro_2.12:2.4.4 ").append(" -i ").append(commandFile).toString();
```

Comment:
our docker images are still 2.3 spark IIRC.. so this interplays well?

Reply:
Haven't tried to run docker images yet, need some time to get back into shape after 1 month hiatus :)

Reply:
@ezhux Hi.. we just landed the spark 2.4.4 PR #1005 . can we rework this on top of this.. Hopefully, makes life simpler.